### ISSUE TITLE
v.2.12.0 missing Visual C++ 2015 Redistributable

### ISSUE THREAD

**Issue by Atario** [hexchat/hexchat#1643]
After upgrading to v.2.12.0, this happens to me every launch:

> \---------------------------
> hexchat.exe - System Error
> \---------------------------
> The program can't start because VCRUNTIME140.dll is missing from your computer. Try reinstalling the program to fix this problem. 
> \---------------------------
> OK   
> \---------------------------

Installing [the appropriate package from Microsoft](https://www.microsoft.com/en-us/download/details.aspx?id=48145) fixes the problem, but the installer should do this for me.

**Comment by TingPing** [comment]
Did you use portable mode?

**Comment by Atario** [comment]
Yes.

**Comment by TingPing** [comment]
Portable mode by design doesn't modify your system, and as such would not install a system dll.

**Comment by Atario** [comment]
I can see your point; however, leaving the user in a nonfunctioning state with no explanation or anything seems pretty bad.  Either a copy of the redistributable should be placed alongside the executable, or else an explanation and link for the user to do something himself should be presented.

**Comment by TingPing** [comment]
The download website does link to the redist but it could be possible for the installer to pop up a dialog about it. We would rather not bundle libraries ourselves whenever possible though.

**Comment by Arnavion** [comment]
Should just remove portable mode from the installer. Make it a separate ZIP download on the page and put "btw you need the redist too from <link>" next to it.

**Comment by ajstadlin** [comment]
I just ran into this problem with Release Git Portable 2.12.2 packaged as PortableGit-2.12.2.2-64-bit.7z.exe.

The vcruntime140.dll dependency appears to be in the *.7z.exe extractor executables, not the Git Portable itself.  

When I extract the contents of the PortableGit-2.12.2.2-64.7z.exe (using 7-zip) and simply use the contents, I don't need the vcruntime140.dll. 

If I want to install the VC2015 redistributable, the 32bit **vc_redist.x86.exe** version works for  PortableGit-2.12.2.2-64.7z.exe, the vc_redist.x64.exe does not.  I need to install **vc_redist.x86.exe** (32bit) to run the PortableGit-2.12.2.2-64-bit.7z.exe extractor.  After extracted, I can uninstall vc_redist.x86.ex and portable git seems to run just fine! 

I also noticed that installing the default Windows Git installation version does not make a difference because it does not install the vcruntime140.dll.

My conclusion is that Git Portable is still portable!  The portability of the 7z extractor is no longer portable if the system does not have the vcruntime140.dll installed.
